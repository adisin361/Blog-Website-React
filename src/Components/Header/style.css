.header {
  background-color: black;
  height: 10vh;
  color: aliceblue;
  font-family: Arial, Helvetica, sans-serif;
  display: flex;
  flex-direction: column;
  justify-content: flex-end;
  padding-left: 19vw;
  padding-right: 19vw;
}

.main-heading {
  font-size: x-large;
}

.nav-bar {
  width: 20vw;
  display: flex;
  justify-content: space-between;
  padding-bottom: 10px;
  align-self: flex-end;
}

.nav-bar a {
  text-decoration: none;
  color: aliceblue;
}