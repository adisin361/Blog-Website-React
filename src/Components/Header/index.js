import * as React from 'react';
import './style.css';
const Header = () => {
  return (
    <>
      <div className='header'>
        <p className='main-heading'>The Artifact</p>
        <p className='sub-heading'><em>Culture & Art blog</em></p>
        <div className='nav-bar'>
          <a href="">Blog</a>
          <a href="">About</a>
          <a href="">Contact</a>
        </div>
      </div>
    </>
  );
};

export default Header;