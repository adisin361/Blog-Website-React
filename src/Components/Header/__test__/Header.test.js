import "@testing-library/jest-dom";
import { fireEvent, render } from "@testing-library/react";
import React from "react";
import Header from '..';

describe("Header", () => {
  it("should match the snapshot", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});