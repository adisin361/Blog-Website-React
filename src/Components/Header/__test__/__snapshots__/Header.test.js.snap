// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <p
      class="main-heading"
    >
      The Artifact
    </p>
    <p
      class="sub-heading"
    >
      <em>
        Culture & Art blog
      </em>
    </p>
    <div
      class="nav-bar"
    >
      <a
        href="#blog"
      >
        Blog
      </a>
      <a
        href="#about"
      >
        About
      </a>
      <a
        href="#contact"
      >
        Contact
      </a>
    </div>
  </div>
</DocumentFragment>
`;
